cloudfront/wellcomelibrary.org/edge-lambda/src/wellcomeLibraryPassthru.test.ts

```diff
@@ -34,14 +34,17 @@ test('http requests are redirected to https', () => {
   );
 });
 
-test('rewrites the host header if it exists', async () => {
-  const request = testRequest('/', undefined, {
+test('redirects wellcomelibrary.org passthrus to wellcomecollection.org', async () => {
+  const request = testRequest('/unknown', undefined, {
```

Comment:
What happens if I do hit `wellcomelibrary.org` after this update? Where will I be redirected then?

Is it worth keeping the existing test, and adding this behaviour as a new test?

Reply:
This test was in place for this bit of code:
https://github.com/wellcomecollection/platform-infrastructure/pull/204/files#diff-ce2a66225bb7a312c1db2c4434c46c33e11141de76621ea6a370bd33df51d45cL18. 

We've removed that so can't test it.

When hitting `/` you will still be redirected to `wc.org/` because we have no other redirect functionality for it. Does it feel like we need something specific for `/` as it's a special URL even though it just falls back to this?

Reply:
Ah, the change to request `/unknown` made this unclear – I thought that was changed because it didn't affect the `/` behaviour.

Reply:
Yea, sorry I should have mentioned I repurposed the test for a completely different test.